lms/djangoapps/student_account/test/test_views.py

```diff
@@ -502,12 +504,29 @@ def test_context(self):
 
     def test_view(self):
 
-        view_path = reverse('account_settings')
-        response = self.client.get(path=view_path)
+        response = self.client.get(path=self.view_path)
 
         for attribute in self.FIELDS:
             self.assertIn(attribute, response.content)
 
+    def test_header_with_programs_listing_enabled(self):
+        """
+        Verify that tabs header will be shown while program listing is enabled.
+        """
+        self.create_programs_config(program_listing_enabled=True)
+        response = self.client.get(path=self.view_path)
+
+        self.assertContains(response, "Programs")
+
+    def test_header_with_programs_listing_disabled(self):
+        """
+        Verify that nav header will be shown while program listing is disabled.
+        """
+        self.create_programs_config(program_listing_enabled=False)
+        response = self.client.get(path=self.view_path)
+
+        self.assertContains(response, "Explore courses")
```

Comment:
Same comment here to have the assert on CSS class rather than a simple string that can generate other unintended matches